sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateWorksetDao.java

```diff
@@ -146,7 +146,9 @@ public String propertyForSortColumn(String original) {
 
   @Override
   public String propertyForUserName(Criteria criteria, boolean creator) {
-    return null;
+    criteria.createAlias("creator", "creator");
+    criteria.createAlias("lastModifier", "lastModifier");
```

Comment:
`"creator"` and `"lastModifier"` should be added to `STANDARD_ALIASES` instead of creating the aliases here